dora/stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -87,6 +87,10 @@ public final class WorkerBenchParameters extends FileSystemParameters {
       description = "If true, skip the data file creation")
   public boolean mSkipCreation = false;
 
+  @Parameter(names = {"--slice-length"},
+      description = "The granularity of coarse data point slices")
```

Comment:
```suggestion
      description = "There will be too many I/O operations during the test, so instead of keeping one data point for each operation, the I/O performed in a small window will be tracked in one result. This argument sets the size of that window.")
```